### ISSUE TITLE
using test_mode in main.py

### ISSUE THREAD

**Issue by programehr** [yjxiong/tsn-pytorch#73]
Hi
It seems that the `main()` functions can work in two modes, which is determined by `args.evaluate`. As I understand, if `args.evaluate` is `True`, then the model must be treated in the eval mode.
If this is correct, shouldn't the validation set be created with `test_mode = True`?
